src/Sulu/Bundle/PreviewBundle/Preview/Exception/TwigException.php

```diff
@@ -19,8 +19,8 @@
 class TwigException extends PreviewRendererException
 {
     /**
-     * @param int $object
-     * @param mixed $id
+     * @param mixed $object
+     * @param string $id
```

Comment:
I think the id is depending on the object behind and so can be `int|string`

Reply:
I always choose the wrong branch :sweat_smile: 

If I change that in the TwigException I should change the PreviewRendererException as well. I just made the PHPDoc match but didn't look at it what could be really inserted.

Reply:
Yes think the `PreviewRendererException` also should accept id as string|int.

The branch need to be rebased that it can be merged to release/2.0

Reply:
The force was strong with me and I rebased successfully.

And I checked all exceptions that they now use the same int|string $id and mixed $object.